cmd/influx_inspect/buildtsi/util.go

```diff
@@ -0,0 +1,218 @@
+package buildtsi
+
+import (
```

Comment:
I feel like this code could just live in the same `buildtsi.go` file.

Reply:
Done. I disagree, but my real problem is that this lives inside a tool, not inside a library package. I originally tried to move this to the `tsi1` package, but that creates a recursive dependency.